Example/Storage/Tests/Unit/FIRStorageMetadataTests.m

```diff
@@ -244,6 +244,55 @@ - (void)testMetadataCopy {
   XCTAssertEqualObjects(metadata0, metadata1);
 }
 
+- (void)testMetadataEmptyUpdate {
```

Comment:
Consider renaming test methods for `updatedMetadata` to something like:

testUpdatedMetadataEmpty
testUpdatedMetadataNonEmpty
testUpdatedMetadataDelete

I was super confused because I expected us to actually be calling `ref updateMetadata` rather than simply testing the update metadata methods.